framework/include/utils/MooseUtils.h

```diff
@@ -88,11 +88,20 @@ std::vector<std::string> split(const std::string & str, const std::string & deli
 
 /**
  * Python like join function for strings.
-
  */
 template <typename T>
 std::string join(const T & strings, const std::string & delimiter);
 
+/**
+ * Check if file exists.
+ */
+bool file_exists(const std::string & filename);
```

Comment:
We already have this `checkFileReadable()`. Also the naming conventions are wrong.